lib/dialects/abstract/query-generator.js

```diff
@@ -2016,6 +2020,8 @@ const QueryGenerator = {
       '<@': '$contained'
     };
 
+    console.log(this._dialect.name);
```

Comment:
Yeh, I noticed that when the logs where spamming. It will be removed in my next commit.